content/modules/rest/pages/business-logic.adoc

```diff
@@ -1,20 +1,208 @@
 = Business Logic
 
-== Service Invocation
+When interacting with the REST API it is oftentimes needed to have an application-level layer of business logic in place that represents the Invocation point for the API. It can be used for orchestration, validation, or other tasks that should happen when the API Client interacts with the system. The xref:rest:entities-api.adoc[Entities API] does not allow placing additional orchestration business logic in place. The Client in this case directly interacts with the entity layer.
```

Comment:
"the Invocation point, the API Client" - we usually don't use capitalization if it's not a proper noun or an object in code.
There are other cases below: "Service, Service Interface". 
Plz remove capitalization in these words, it's better for uniformity and IMHO for readability too.

Reply:
done